modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/exec/ExchangeServiceImpl.java

```diff
@@ -114,9 +114,7 @@ public CompletableFuture<Void> sendError(String nodeName, UUID queryId, long fra
 
         if (!(traceableErr instanceof TraceableException)) {
             traceableErr = error = new IgniteInternalException(INTERNAL_ERR, error);
-        }
 
-        if (!(traceableErr instanceof QueryCancelledException)) {
```

Comment:
why did you merge these branches?

Reply:
To avoid excessive logging. For example `Division by zero` in query was here.